### DISCUSSION TITLE
FreeBSD support

### DISCUSSION THREAD

**Discussion by ericLemanissier** [turtlebrowser/turtlebrowser#54] [General]
Just trying this new github feature, so I'm reposting my twitter DM:

there seem to remain only two blocking errors on freebsd:
ERROR: Feature 'zstd' was enabled, but the pre-condition 'libs.zstd' failed.

ERROR: Feature 'system-sqlite' was enabled, but the pre-condition 'features.sql-sqlite && libs.sqlite3' failed.

they should be solved with https://github.com/conan-io/conan-center-index/pull/3832 and https://github.com/conan-io/conan-center-index/pull/3831

**Comment by ericLemanissier** [comment]
@patricia-gallardo

The two PR mentionned in my last message are now merged. the BSD build should be able to go one step further.

**Comment by patricia-gallardo** [comment]
I’ll trigger a new build!

**Reply by patricia-gallardo** [reply]
It should probably also use gmake - which is the regular gnu make iiuc:
"make: don't know how to make j1. Stop" https://github.com/turtlebrowser/turtlebrowser/runs/1539377299?check_suite_focus=true

**Comment by ericLemanissier** [comment]
yes, I'm trying https://github.com/bincrafters/conan-qt/commit/ef37dfd81fbbcd340adbc85b24f9129d0cc8100a

**Reply by ericLemanissier** [reply]
I'm doing some tests here https://github.com/ericLemanissier/tsteeper/actions?query=workflow%3AFreeBSD

**Reply by patricia-gallardo** [reply]
A bit meta (since I’m not used to this discussion feature myself) I don’t really know where to “answer” 😅 anyhow, did you see the Qt location error? Is that a missing dependency?

**Reply by ericLemanissier** [reply]
yes `error: use of undeclared identifier 'SCHED_IDLE'`, it seems the kernel is too old (?). I am not sure what we should do here

**Comment by patricia-gallardo** [comment]
It’s building Qt! 🥳
